base.sh

```diff
@@ -446,19 +444,19 @@ endofjob () {
     myecho
     myecho "begin $fun `date`"
     # Add stdout and stderr to central files
-    if [ -f $chkpoststage ]
+    if [ -f "$chkpoststage" ]
     then
-       cp -a $chkpoststage /tmp/poststage
+       cp -a "$chkpoststage" /tmp/poststage
```

Comment:
Maybe do the same with the quote remarks, and just merge this one? I'm fine with that.